action.yml

```diff
@@ -0,0 +1,59 @@
+runs:
+  using: 'docker'
+  image: 'Dockerfile'
+inputs:
+  repo:
+    description: "The repo to run against"
+    required: false
+  instance_count:
+    description: "The number of instances to create"
+    required: false
+    default: '1'
+  gh_pat:
+    description: "The GHA PAT"
+    required: true
+  action:
+    description: 'Whether to start or stop. Options: "start", "stop"'
+    required: true
+  provider:
+    description: 'The cloud provider to use to provision a runner. Example: "aws"'
+    required: true
+  gh_runner_labels:
+    description: 'The comma seperated list of runner labels for GitHub when stopping'
+    required: false
+  instance_mapping:
+    description: 'A JSON object mapping instance ids to unique GH runner labels'
+    required: false
+  aws_instance_ids:
+    description: 'The comma seperated list of ids for AWS when stopping'
+  aws_image_id:
+    description: "The machine AMI to use for your runner"
+    required: false
```

Comment:
again, more details on requirements (or lack thereof) for the AMI would be good here

Reply:
This has been resolved in [3e92105](https://github.com/omsf-eco-infra/gha-runner/pull/8/commits/3e92105d139d4658013492e6c6e8980d54b23a0d)